plugins/auth-backend-module-cloudflare-access-provider/src/authenticator.ts

```diff
@@ -38,7 +38,7 @@ export function createCloudflareAccessAuthenticator(options?: {
     async defaultProfileTransform(result: CloudflareAccessResult) {
       return {
         profile: {
-          email: result.claims.email,
+          email: result.claims.email ?? result.cfIdentity.email,
```

Comment:
Will `cfIdentity` always contain the correct email, should we use that instead? Always nice to be explicit in this area 😁

Reply:
Good point. As far as I can tell, yes, the `cfIdentity` struct will contain the email in both cases so that's simpler.